### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by Zeitsperre**

### CODE REVIEW

File: cookiecutter.json [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
-	"full_name": "Taylor \"Nekroze\" Lawson",
-	"email": "nekroze@eturnilnetwork.com",
-	"github_username": "Nekroze",
-	"project_name": "A Lot of Effort",
-	"repo_name": "alotofeffort",
-	"project_short_description": "Deploy static HTML sites to S3 with the simple 'alotofeffort' command.",
-	"release_date": "2013-07-10",
-	"year": "2013",
-	"version": "0.1.0"
+	"full_name": "Trevor James Smith",
```
Comment (Nekroze) [inline cookiecutter.json:2]: These kinds of changes should remain in your fork, although that will not cause an issue to merge this in.

File: README.rst [hunk 2]
```diff
@@ -1,14 +1,16 @@
-======================
-cookiecutter-pypackage
-======================
+==============================
+cookiecutter-pypackage-ouranos
```
Comment (Nekroze) [inline README.rst:2]: I have not changed the name of the repo, this might be confusing.

File: README.rst [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
+==============================
 
-Cookiecutter template for a Python package. See https://github.com/audreyr/cookiecutter.
+Cookiecutter template (Ouranos recipe) for a Python package. See https://github.com/audreyr/cookiecutter.
```
Comment (Nekroze) [inline README.rst:5]: This should probably be excluded from the merge to my repo, just to keep the delineation clear.

File: {{cookiecutter.repo_name}}/CONTRIBUTING.rst [hunk 4]
```diff
@@ -13,7 +13,7 @@ Types of Contributions
 Report Bugs
 ~~~~~~~~~~~
 
-Report bugs at https://github.com/{{ cookiecutter.github_username }}/{{ cookiecutter.repo_name }}/issues.
+Report bugs at https://github.com/{{ cookiecutter.github_organization }}/{{ cookiecutter.repo_name }}/issues.
```
Comment (Nekroze) [inline {{cookiecutter.repo_name}}/CONTRIBUTING.rst:16]: I see the reason for this but either this way or the old way we are excluding someone, can we instead make this `github_namespace` that cookiecutter asks for instead?

File: LICENSE [hunk 5]
```diff
@@ -0,0 +1,26 @@
+Copyright (c) Audrey Roy Greenfeld and individual contributors.
```
Comment (Nekroze) [inline LICENSE:1]: Would this license require users of this template to provide a copy of this license text along with the dynamically generated one in order to be compliant?

**Review by Nekroze — CHANGES_REQUESTED** [review]

**Comment by Nekroze** [comment]
Hi @Zeitsperre 

Thank you for this contribution, there are some great improvements there that I and this repos users will like I think. Much appreciated. I have just left a few comments in places I have some questions or concerns before we merge this.

**Comment by Zeitsperre** [comment]
Hi, please excuse the confusion!

I've been working off of a fork of yours for the organization I work with and was performing some changes to that fork. If you're interested in some of my changes though, I can re-fork this repo on my personal account certainly merge some of the more relevant ones towards yours and update yours accordingly.

**Comment by Nekroze** [comment]
No worries @Zeitsperre, its great stuff and I would love to merge it. I am just concerned with the license and copyright changes/additions at present, I am not super comfortable merging this as is and then removing them or genericizing some stuff again. I will try to get some time this weekend to fork your branch and make those adjustments if that helps.

**Comment by Zeitsperre** [comment]
Hi @Nekroze, I'm archiving this PR and moving the code under a personal account. I'll be able to generalize it a bit, making changes as needed there. Any interest in a fresher PR ?

**Comment by Nekroze** [comment]
apologies for the late reply, been moving house and this notification got lost along the way. If you have a more generic version @Zeitsperre that would be fantastic to PR, it is still on my todo list but its quite far down these days sadly.

**Pull request closed without merging**